src/views/TradingCompetition/components/YourScore/UserRankBox.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, Text, FlexProps } from '@pancakeswap-libs/uikit'
+
+interface UserRankProps extends FlexProps {
+  title?: string
+  footer?: string
+}
+
+const Wrapper = styled(Flex)`
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid ${({ theme }) => theme.colors.inputSecondary};
+  border-radius: ${({ theme }) => theme.radii.default};
+  padding: 8px 8px;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    padding: 8px 18px;
+  }
+
+  ${({ theme }) => theme.mediaQueries.md} {
+    padding: 8px 24px;
```

Comment:
Since you're just modifying the left and right margin I think it is better to be explicit.

E.g.
```
padding-left: 18px;
padding-right: 18px;
```